php/commands/sites.php

```diff
@@ -264,7 +264,8 @@ public function aliases($args, $assoc_args) {
     $this->log()->info($message);
 
     if ($print) {
-      eval(str_replace(array('<?php', '?>'), '', $content));
+      //FIXME
+      //eval(str_replace(array('<?php', '?'), '', $content));
```

Comment:
Yes, it does. I commented it out because it messes with syntax highlighting. Thanks!